src/main/resources/wa-task-initiation-employment-et_englandwales.dmn

```diff
@@ -2179,6 +2179,329 @@ else null</text>
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0rqkkey">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_04ae2wj">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1qn09hb">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1qrj7gt">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_08ko27s">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0ajw4rn">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_189va9n">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1cd3ao4">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1iuvq4q">
+          <text>"Change my personal details"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1d0e98d">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0tn4k7b">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_101wv5s">
+          <text></text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1ypt0z2">
+          <text>"AmendClaimantDetails"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0p3xjto">
+          <text>"Amend Party Details"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_01u5xs1">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0np3z8l">
+          <text>"Amendments"</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_0ige02e">
+        <inputEntry id="UnaryTests_1da4okr">
+          <text>"tseClaimantRepResponse"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1cgyz1i">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1wvl1z1">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_08br72a">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_117eg1d">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_18uyxon">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_16yztyn">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0dek4m6">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1lo02uk">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1xkywpp">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1f3jmy8">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1qas7r4">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_08tml81">
+          <text>"Claimant"</text>
```

Comment:
Noticed this when reviewing callbacks - the response from is set to Claimant Representative and not Claimant so this task might not trigger - https://github.com/hmcts/et-ccd-callbacks/pull/2259/files#diff-f007255afc0d633a2c2099ce178ce90cd4e16e7f2684a76a8d24177611e13700R211

Reply:
addressed comments [e95be0d](https://github.com/hmcts/et-wa-task-configuration/pull/263/commits/e95be0d9a145b0fd0fe51e0b1a9ff52b125285c6)